Unix/installbuilder/service_scripts/service_control.unix

```diff
@@ -143,6 +143,7 @@ restart_omi()
 
     if [ "$TYPE" = "A" -o "$TYPE" = "H" -o "$TYPE" = "S" ]; then
 	stop_omi
+	sleep 1
```

Comment:
Blech, I hate this kind of thing. It's obviously a timing problem, which is why we haven't seen it before. What if the system is slow enough where 1 second isn't long enough?

Please actively checking if OMI is down and then wait only if OMI is still running. We've done this sort of thing in other areas of code, please use a similar method here. Thanks.